client/sites-dashboard/components/no-sites-message.tsx

```diff
@@ -8,15 +8,32 @@ const NoSitesLayout = styled( EmptyContent )`
 	flex-direction: column;
 	justify-content: center;
 	align-items: center;
+
+	${ ( { illustration } ) =>
+		! illustration && {
+			marginTop: '10%',
+		} }
+
+	.empty-content__line {
+		margin-bottom: 20px;
+	}
```

Comment:
We can style the `SecondaryText` element (below on that file) instead of nesting selectors here. This way we won't break the componentization of styles.

Reply:
Thanks. I did that initially as I needed to reduce the margin  for `.empty-content__line` from 40px to 20px. It looks like I was able to reduce it to 0 for the `SecondaryText` element instead.